doc/en/user/source/community/cog/index.rst

```diff
@@ -125,19 +125,15 @@ On the table below, replace the "$ALIAS$" template with HTTP or HTTPS or S3 if y
      - Default password (secret access key) for AWS basic authentication credentials
 
 Client configuration (System Properties)
------------------------------------------------------------------
-Note that all the settings reported in the previous tables can also be specified using System Properties instead of Environment variables.
+----------------------------------------
+Note that all the IIO  settings reported in the previous tables can also be specified using System Properties instead of Environment variables.
 You just need to replace UPPER CASE words with lower case words and underscores with dots.
 So, the value for Maximum HTTP requests can be specified by setting either a ``IIO_HTTP_MAX_REQUESTS`` Environment variable or a ``iio.http.max.requests`` Java System Property alternatively (Environment variables are checked first).
 
+By default, when accessing a COG, a first chunk of 16 KB is read in attempt to parset the header so that the reader will know offset and length of the available tiles. When dealing with big files, there might be the case that the whole header won't fit in the first chunk so additional reads (chunks of the same size) will be progressively made to complete the parsing.
```

Comment:
```suggestion
By default, when accessing a COG, a first chunk of 16 KB is read in attempt to parset the header so that the reader will know offset and length of the available tiles. When dealing with files hosting many tiles, it might happen that the whole header won't fit in the first chunk. In this case additional reads (chunks of the same size) will be progressively made to complete loading the header.
```